printer_mods/hbo/Voron0_Skirts/Readme.md

```diff
@@ -0,0 +1,86 @@
+# Collection of Voron 0 Skirts
+
+
+
+
+This is a Collection of Voron 0 skirts that provide some additional features and bring the asthetics closer to the bigger VORON brothers.
+
+![Printed Panel](Images/VORON_V0.1_2022.png)
+
+Right now there are 3 variations for the and one commmon rear.
+To start with the rear ..
+
+## Rear Skirts
+
+The rear skirts match all the different front skirts and have the following features
+
+- integrated power inlet mount. To be used with the "standard" v0 power inlet
+- 2 keystone mount to get acces to the raspery ports
+- bowden clip mount support. Disigned for the E3D style bowden clips. This allows to push the fillament and not the tube from the outside. Read for the rabit feader :)
```

Comment:
`Disigned` -> `Designed`
`fillament` -> `filament`
`rabit feader` -> `rabbit feeder`  (Consider calling this by it's full name "Enraged Rabbit Carrot Feeder", or just ERCF.

Reply:
fixed